scripts/devops_tasks/setup_execute_tests.py

```diff
@@ -262,6 +270,9 @@ def execute_global_install_and_test(
     else:
         extended_pytest_args.extend(["--durations=10", "--cov", "--cov-report="])
 
+    if args.xdist:
+        extended_pytest_args.extend(["-n", "8"])
```

Comment:
Why a constant instead of `auto`?

Reply:
`auto` sets the parallelization at the number of logical CPUs on the machine. Because these tests are largely network bound (and almost never CPU bound) it makes sense to increase parallelism beyond the number of cores on the the Azure DevOps VM (2 cores) to complete tests faster.